lib/renderer.coffee

```diff
@@ -24,25 +45,12 @@ exports.toHtml = (text='', filePath, grammar, callback) ->
   roaster text, options, (error, html) =>
     return callback(error) if error
 
-    grammar ?= atom.grammars.selectGrammar(filePath, text)
-    # Default code blocks to be coffee in Literate CoffeeScript files
-    defaultCodeLanguage = 'coffee' if grammar.scopeName is 'source.litcoffee'
-
     html = sanitize(html)
     html = resolveImagePaths(html, filePath)
-    html = tokenizeCodeBlocks(html, defaultCodeLanguage)
-    callback(null, html.html().trim())
-
-exports.toText = (text, filePath, grammar, callback) ->
-  exports.toHtml text, filePath, grammar, (error, html) ->
-    if error
-      callback(error)
-    else
-      string = $(document.createElement('div')).append(html)[0].innerHTML
-      callback(error, string)
+    callback(null, html.trim())
 
 sanitize = (html) ->
-  o = cheerio.load("<div>#{html}</div>")
+  o = cheerio.load(html)
```

Comment:
We needed to remove the step where the DOM was converted back to a string at the very end in order to insert `atom-text-editor` nodes successfully. When I tried to do this, I got really confused by the signature of the various stages of the pipeline. The naming seemed to imply they were string -> string, but that didn't end up being the case, so I tried to iron out the types to clarify it and this was causing a div to be inserted in the output, which was breaking my tests. I was able to remove it without causing test failures, so I assumed my ironing out of the pipeline made it unnecessary.